receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md

```diff
@@ -12,8 +12,11 @@ These are the metrics available for this scraper.
 | **process.disk.io** | Disk bytes transferred. | By | Sum(Int) | <ul> <li>direction</li> </ul> |
 | **process.disk.io.read** | Disk bytes read. | By | Sum(Int) | <ul> </ul> |
 | **process.disk.io.write** | Disk bytes written. | By | Sum(Int) | <ul> </ul> |
+| process.involuntary_ctx_switches | Number of times the process has been involuntarily context switched. | {context switches} | Sum(Int) | <ul> </ul> |
 | **process.memory.physical_usage** | The amount of physical memory in use. | By | Sum(Int) | <ul> </ul> |
 | **process.memory.virtual_usage** | Virtual memory size. | By | Sum(Int) | <ul> </ul> |
+| process.open_file_descriptors | Number of file descriptors in use by the process. | {file descriptors} | Sum(Int) | <ul> </ul> |
```

Comment:
In the filesystem is there an equivalent? How do we name it there?

Reply:
File descriptors don't necessarily point to anything within the filesystem (e.g. `stdout` has a file descriptor ID of `1`in POSIX systems), and I don't see an similar metric, so I don't believe there is an equivalent. Please let me know if I've misunderstood.